bgpd/bgpd.c

```diff
@@ -232,8 +232,18 @@ void bgp_option_norib_set_runtime(void)
 	zlog_info("Disabled BGP route installation to RIB (Zebra)");
 
 	for (ALL_LIST_ELEMENTS_RO(bm->bgp, node, bgp)) {
-		FOREACH_AFI_SAFI(afi, safi)
+		FOREACH_AFI_SAFI (afi, safi) {
+			/*
+			 * Stop a crash, more work is needed
+			 * here to properly add/remove these types of
+			 * routes from zebra.
+			 */
+			if (safi == SAFI_EVPN || safi == SAFI_MPLS_VPN ||
```

Comment:
nit: Maybe we can reuse `bgp_fibupd_safi()`?

Reply:
sure didn't know that existed.

Reply:
fixed